lib/main.ts

```diff
@@ -655,3 +719,37 @@ export async function validateNonce(req: NextApiRequest) {
     throw `Nonces are not equal!`;
   }
 }
+
+export async function fetchRecords(repository: Repository<Confession>) {
+  let repo = repository;
+  let recordsFetched;
+  console.log("Iterating through the records!");
+  try {
+    const now = new Date();
+    const weekAgo = now.setDate(now.getDate() - 7) / 1000;
+    recordsFetched = await repository.find({
+      where: [
```

Comment:
May also be a good idea to filter out confessions that already have an empty hash/salt set